source/NVDAObjects/IAccessible/winword.py

```diff
@@ -258,14 +279,25 @@ def script_setRowHeader(self,gesture):
 			else:
 				# Translators: a message reported in the SetRowHeader script for Microsoft Word.
 				ui.message(_("Cannot find row {rowNumber} column {columnNumber}  in row headers").format(rowNumber=cell.rowIndex,columnNumber=cell.columnIndex))
-	script_setRowHeader.__doc__=_("Pressing once will set this cell as the first row header for any cells lower and to the right of it within this table. Pressing twice will forget the current row header for this cell.")
 
+	@script(

```

Comment:
can/should these scripts added in this document get descriptions added?

Reply:
Hello,
I am not someone who is proficient in Word, and I do not know their function.
I have only converted the old syntax to new and added the scripts with a description to the correct category.
To complete the descriptions, I will leave it to someone more experienced in Word to do so.